### DISCUSSION TITLE
High memory usage by using the offline mode for my whole s3 bucket

### DISCUSSION THREAD

**Discussion by DiesDasJenes** [iterate-ch/cyberduck#15870] [Q&A] [answered]
Hi,
I thought for better usage and search offline I could tell Mountain Duck to synchronize the complete content of the bucket I connected to. Now 10GB(90%) of memory is going to Mountain Duck and I don't know how I can tell the application to stop doing that. Anyone knows what to do now? Would re-installing help here or would it pick the last command up again?

**Comment by dkocher** [comment, marked as answer]
To cancel the download of files currently synchronizing because of having selected _Keep Offline_ follow these steps:

1. [Pause Synchronization](https://docs.mountainduck.io/mountainduck/connect/sync/#pause-sync) 
2. Choose [_Delete on local disk_](https://docs.mountainduck.io/mountainduck/connect/sync/#delete-on-local-disk) from the context menu
3. Resume Synchronization

**Comment by DiesDasJenes** [comment]
Thanks for sharing.
